spec/factories/loans.rb

```diff
@@ -55,11 +55,12 @@
 
 
     trait :active do
-      status_option_id Loan::STATUS_OPTIONS.value_for('Active')
+      # note: can't seem to count on the seed.db data having been loaded when these are executed
+      status_option_id 1 #Loan.status_option_set.value_for_label('Active')
```

Comment:
The factories shouldn't have any dependencies on the `seeds.rb` data, ideally. What I'd like to see is something like

```ruby
after(:create) do |loan|
  option_set = create(:option_set, model_type: 'Loan', model_attribute: 'status')
  create(:option, value: 'active', option_set: option_set)
  create(:option, value: 'completed', option_set: option_set)
end

trait :active do
  status_option_id  Loan.status_option_set.options.find_by(value: 'active')
end
```

So that the factory can create everything it needs without relying on any particular database state, but the fact that it's so hard to query through this structure for pretty routine needs is making me kind of apprehensive about the approach, so I sort of want to think on this more.